src/Dev/Deprecation.php

```diff
@@ -223,29 +256,82 @@ public static function set_enabled($enabled)
         // noop
     }
 
+    /**
+     * If true, any E_USER_DEPRECATED errors should be treated as coming
+     * directly from this class.
+     */
+    public static function triggeringError(): bool
+    {
+        return self::$triggeringError;
+    }
+
+    /**
+     * Determine whether deprecation warnings should be suppressed in HTTP responses.
+     *
+     * This will be overriden by the SS_DEPRECATION_SUPPRESS_HTTP environment variable if present.
+     */
+    public static function setShouldSuppressForHttp(bool $value): void
```

Comment:
```suggestion
    public static function setShowForHttp(bool $value): void
```
Need to update the other 'setShowSuppress' methods to

Reply:
done